storybook/stories/_support/config.js

```diff
@@ -44,7 +44,7 @@ export const osNames: Array<any> = Object.keys(operatingSystems);
 export const osMinWindowHeights = {
   Windows: '541px',
```

Comment:
@DeeJayElly you forgot to update the min-heights for Windows an Linux...

Reply:
Done